### PR TITLE
[SPARK-45807][SQL] Return View after calling replaceView(..)

### PR DESCRIPTION
**Pull request by nastra**
### What changes were proposed in this pull request?

Follow-up API improvements based on from https://github.com/apache/spark/pull/43677

### Why are the changes needed?

Required for DataSourceV2 view support.

### Does this PR introduce _any_ user-facing change?

No
### How was this patch tested?

N/A

### Was this patch authored or co-authored using generative AI tooling?

N/A

### CODE REVIEW

**Comment by nastra** [comment]
@cloud-fan @jzhuge since you both looked at https://github.com/apache/spark/pull/43677 in the past, could you also take a look at this one please?

File: sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/ViewCatalog.java [hunk 1]
```diff
@@ -129,10 +129,11 @@ default boolean viewExists(Identifier ident) {
    *
    * @param viewInfo the info class holding all view information
    * @param orCreate create the view if it doesn't exist
+   * @return the created/replaced created
```
Comment (cloud-fan) [inline sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/ViewCatalog.java:132]: shall we follow https://github.com/apache/spark/commit/d8fb91e61352e57e733e7d7e4978c8ce555454b1#diff-76ba8535886252d11f5de158f6e465ea92dc07f3e2b3f6a976a5cb412fb19112R190 ?

Comment (cloud-fan) [reply sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/ViewCatalog.java:132]: Yea only change the doc for now. We will need to deal with null when implementing the v2 view commands.

Comment (nastra) [reply sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/ViewCatalog.java]: I've updated the doc for create + replace view

**Review by cloud-fan — APPROVED** [review]

**Comment by cloud-fan** [comment]
The test failure is unrelated, thanks, merging to master!

**Pull request closed without merging**